src/Microsoft.ML.Transforms/ExtensionsCatalog.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML
+{
+    public static class MissingValueIndicatorCatalog
+    {
+        /// <summary>
+        /// Initializes a new instance of <see cref="MissingValueIndicatorEstimator"/>
+        /// </summary>
+        /// <param name="catalog">The categorical transform's catalog.</param>
+        /// <param name="columns">The names of the input columns of the transformation and the corresponding names for the output columns.</param>
+        public static MissingValueIndicatorEstimator IndicateMissingValues(this TransformsCatalog catalog, params (string inputColumn, string outputColumn)[] columns)
```

Comment:
>Indicate [](start = 53, length = 8)

'indicate' seems weird. I don't have a better suggestion though...

Reply:
that should be fine. sklearn uses the same terminology: [http://scikit-learn.org/stable/modules/generated/sklearn.impute.MissingIndicator.html#sklearn.impute.MissingIndicator](http://scikit-learn.org/stable/modules/generated/sklearn.impute.MissingIndicator.html#sklearn.impute.MissingIndicator)

---
In reply to: [229778482](https://github.com/dotnet/machinelearning/pull/1460#discussion_r229778482) [](ancestors = 229778482)